internal/services/mssqlmanagedinstance/mssql_managed_instance_failover_group_resource.go

```diff
@@ -118,6 +119,16 @@ func (r MsSqlManagedInstanceFailoverGroupResource) Arguments() map[string]*plugi
 				},
 			},
 		},
+
+		"secondary_type": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Default:  string(instancefailovergroups.SecondaryInstanceTypeGeo),
+			ValidateFunc: validation.StringInSlice([]string{
+				string(instancefailovergroups.SecondaryInstanceTypeGeo),
+				string(instancefailovergroups.SecondaryInstanceTypeStandby),
+			}, false),
```

Comment:
We could use the function from the sdk here, what do you think?
```suggestion
			ValidateFunc: validation.StringInSlice(instancefailovergroups.PossibleValuesForSecondaryInstanceType(), false),
```

Reply:
oh thanks i didn't know about it